### PR TITLE
Aden 4639

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [MobileAdsDataProvider.java#L158](https://github.com/Wikia/selenium-tests/blob/5b8608dc5b253e57e1062d50a3b42983c65af0f9/src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java#L158): Define a constant instead of duplicating this literal "\"loc\":\"top\"" 8 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestPad.java [hunk 1]
```diff
@@ -1,73 +0,0 @@
...
-      "InstantGlobals.wgSitewideDisablePaidAssetDrop=1";
-
-  @InBrowser(
-      browser = Browser.CHROME,
-      emulator = Emulator.GOOGLE_NEXUS_5
-  )
-  @Test(
-      dataProviderClass = AdsDataProvider.class,
-      groups = "TestPadMercury",
-      dataProvider = "testPad"
```
Comment (mleczey) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestPad.java]: Can we remove data providers as well here?

Comment (bulaszenko) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestPad.java]: Done

**Comment by dianafa** [comment]
LGTM

**Pull request merged**